src/Web/Streams/WritableStreamController.purs

```diff
@@ -0,0 +1,9 @@
+module Web.Streams.WritableStreamController where
+
+import Effect (Effect)
+import Effect.Exception (Error)
+import Prelude (Unit)
+
+foreign import data WritableStreamController :: Type -> Type
+
```

Comment:
Looks like the readonly attribute `signal` is missing here. If it's not added in this PR, we should open an issue tracking that.

Reply:
(see https://github.com/purescript-web/purescript-web-streams/pull/9/commits/ae0a869244b5253af2861665c3e2efef3baaccf0#r919656956)

Reply:
Ah; it appears that the returned `AbortSignal` type is currently defined in the [web-fetch](https://pursuit.purescript.org/packages/purescript-web-fetch/3.0.0/docs/Web.Fetch.AbortController#t:AbortSignal) package, and it needs be moved to somewhere more centralized to avoid a circular dependency. A new "web-abort-signal" package seems somewhat overkill, but given its [inclusion in the DOM standard](https://dom.spec.whatwg.org/#abortsignal) rather than the fetch or stream standard I'm not sure what the best course of action would be. I'll leave this out of the next revision for now.